jobs/jobs.go

```diff
@@ -191,7 +196,7 @@ func GetEmailTemplate(htmlBody string, title string, attachmentBytes *bytes.Buff
 	msg.SetHeader("Subject", title)
 	msg.SetBody("text/html", htmlBody)
 
-	msg.Attach("resume.pdf", gomail.SetCopyFunc(func(w io.Writer) error {
+	msg.Attach("resume"+Extension, gomail.SetCopyFunc(func(w io.Writer) error {
```

Comment:
I think it's `"resume."+Extension`

Reply:
No, we used `filepath.Ext(fileHeader.Filename)` so this method return with `dot extension`